src/sentry/api/endpoints/team_details.py

```diff
@@ -14,13 +15,23 @@
 from sentry.api.serializers import serialize
 from sentry.api.serializers.models.team import TeamSerializer as ModelTeamSerializer
 from sentry.api.serializers.rest_framework.base import CamelSnakeModelSerializer
+from sentry.apidocs.constants import (
+    RESPONSE_FORBIDDEN,
+    RESPONSE_NO_CONTENT,
+    RESPONSE_NOT_FOUND,
+    RESPONSE_UNAUTHORIZED,
+)
+from sentry.apidocs.examples.team_examples import TeamExamples
+from sentry.apidocs.parameters import GlobalParams, TeamParams
 from sentry.models.scheduledeletion import RegionScheduledDeletion
 from sentry.models.team import Team, TeamStatus
 
 
-class TeamSerializer(CamelSnakeModelSerializer):
+@extend_schema_serializer(exclude_fields=["name"])
+class TeamDetailsSerializer(CamelSnakeModelSerializer):
```

Comment:
I had to rename here because open-api was complaining that it couldn't differentiate between this seralizer and `from sentry.api.serializers.models.team import TeamSerializer`

Reply:
Got it. I usually call them request serializer. `TeamRequestSerializer`